apps/fabric-website/src/SiteDefinition/SiteDefinition.pages/Controls/android.tsx

```diff
@@ -9,6 +9,19 @@ export const controlsPagesAndroid: INavPage[] = [
     component: () => <LoadingComponent title="Controls" />,
     getComponent: cb => require.ensure([], require => cb(require<any>('../../../pages/Overviews/ControlsPage/ControlsPage').ControlsPage))
   },
+  {
+    title: 'Basic Inputs',
+    url: '#/controls/android/basic-inputs',
```

Comment:
The parent page for each category typically points to the first control's page:
```suggestion
    url: '#/controls/android/button',
```